code/handlers/ldap.q

```diff
@@ -50,29 +78,32 @@ login:{[user;pass]                                              / validate login
     incache[`time]>.z.p-.ldap.checktime;                        / previous attemp occured within checktime period
     incache[`pass]~np:md5 pass                                  / same password was used
   );
-    1b;
-    @[{.ldap.authenticate[x]`success};dict;0b]                  / attempt authentication
+    enlist[`ReturnCode]!enlist 0i;
+    .[.ldap.bind;(.ldap.sessionID;dict);0b]                 / attempt authentication
```

Comment:
I think the error trap needs to return the same as line above if the bind fails i.e. the one item dictionary, not a boolean

Reply:
When having the error trap return a dictionary, would it be enlist[`ReturnCode]!enlist 0i, if im reading the login function right, that would assign authorised to the dictionary enlist[`ReturnCode]!enlist 0i, which would then allow authentification even if the bind breaks?

Reply:
Good point - yes, should be a non-zero return code, can probably just pick e.g. `-2i` which is "LDAP_LOCAL_ERROR              local error (failed dynamic memory allocation?)" - I think if something fails here we can consider it a "local error".

Reply:
Good job, will add that in now